base/setup/welcome/lang/hi-IN.rc

```diff
@@ -2,7 +2,7 @@
  * PROJECT:     ReactOS Welcome Hindi Translation
  * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
  * PURPOSE:     To make Hindi translation of Welcome
```

Comment:
... because here you can just say "Hindi Translation" (we know it's for "welcome" !)